db/migrate/20231107202222_create_income_details.rb

```diff
@@ -0,0 +1,13 @@
+class CreateIncomeDetails < ActiveRecord::Migration[7.0]
+  def change
+    create_table :income_details, id: :uuid do |t|
+      t.uuid :person_id, null: false
+      t.string :income_above_threshold
+
+      t.timestamps
```

Comment:
Never seen [this](https://apidock.com/rails/ActiveRecord/ConnectionAdapters/Table/timestamps) before, this is handy!

Reply:
Yep pretty useful :)